src/main/kotlin/org/rust/ide/refactoring/extractFunction/ExtractFunctionParameterTablePanel.kt

```diff
@@ -17,6 +23,57 @@ class ParameterDataHolder(val parameter: Parameter, val onChange: () -> Unit) :
     }
 }
 
+class ChooseColumn : ColumnInfo<ParameterDataHolder, JBCheckBox>(null) {
+
+    override fun valueOf(item: ParameterDataHolder): JBCheckBox? = JBCheckBox(null, item.parameter.isSelected).apply {
+        this.horizontalAlignment = JBCheckBox.CENTER
+        this.addChangeListener {
+            item.parameter.isSelected = this.isSelected
+            item.onChange()
+        }
+    }
+
+    override fun getColumnClass(): Class<*> = JBCheckBox::class.java
+
+    override fun isCellEditable(item: ParameterDataHolder?): Boolean = true
+}
+
+class ChooseRenderer : TableCellRenderer {
```

Comment:
Looks like you can use `BooleanTableCellRenderer` instead of introducing your own renderer